draft-jmiller-json-web-proof.md

```diff
@@ -221,7 +221,7 @@ The same JWP using the JSON Serialization:
 }
 ```
 
-After using the BBS+ algorithm to generate an unlinkable proof revealing only the 2nd and 4th payload of the given name and email in this example, the result will be a new JWP with the same Protected Header only those payloads included:
+After using the BBS algorithm to generate an unlinkable proof revealing only the 2nd and 4th payload of the given name and email in this example, the result will be a new JWP with the same Protected Header only those payloads included:
 
 ```
   ImV5SjBlWEFpT2lKS1VGUWlMQ0poYkdjaU9pSkNRbE1ySWl3aVkyeGhhVzF6SWpwYkltWmhiV2xzZVY5dVlXMWxJaXdpWjJsMlpXNWZibUZ0WlNJc0ltVnRZV2xzSWl3aVlXZGxJbDE5Ig
```

Comment:
For readability, I might repeat the protected header, and the entire decoded payload here... just to make the relationship to this encoded value even clearer.

Reply:
The goal is to have the fully unrolled example as part of the JPA BBS definition with only a high level summary here in the main draft.  If there's time I might bring up this strategy on the sync tomorrow as there could definitely be a better approach.